### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by pamlane**
Paul, please review my Streams Runner updates, and merge with IBMStreams if OK.

### CODE REVIEW

File: docs/beamrunner/objstor.md [hunk 1]
```diff
@@ -64,12 +68,12 @@ mvn exec:java -Ps3 \
 
    The command is similar to the one that is used in the README.md for this sample application, but there are a few important differences:
 
-    - The `--filesToStage` option is used to move the local `README.md` file to the runtime environment on IBM Cloud to be used as input for the sample. Alternatively, this file can be uploaded to your Cloud Object Storage service by using the web UI or command-line and referenced with the `s3://` scheme, but staging it this way allows you to use it without that extra step.
+    - The `--filesToStage` option is used to move the local `README.md` file to the runtime environment on IBM Cloud to be used as input for the sample. Alternatively, this file can be uploaded to your Cloud Object Storage service by using the web UI or command-line and referenced with the `s3://` scheme, but staging it by using the  -filesToStage` option eliminates the extra step.
```
Comment (pgerv12) [inline docs/beamrunner/objstor.md]: Does this need to be 
```
`--filesToStage`
```

File: _data/custom-sidenavs.yaml [hunk 2]
```diff
@@ -11,18 +11,17 @@ tags:
...
         - page: "WordCount sample app"
           url: "/docs/beamrunner/wordcount"
-    - page: "I/O options"
-      url: "/docs/beamrunner/io"
-      subpages:
-        - page: "Using IBM Cloud Object Storage"
+        - page: "FileStreamSample sample app"
           url: "/docs/beamrunner/objstor"
-        - page: "Using IBM Cloud Message Hub"
+        - page: "io sample app"
```
Comment (pgerv12) [inline _data/custom-sidenavs.yaml]: Capitalize io?

File: docs/beamrunner/beamrunner-2a-onprem.md [hunk 3]
```diff
@@ -18,68 +18,69 @@ Installing IBM® Streams Runner for Apache Beam from an installation of IBM Stre
 
 ## Before you start
 The on-premises installation of Streams Runner supports Apache Beam 2.4
-applications, so your application must use the [version 2.4 Java
-API](https://beam.apache.org/documentation/sdks/javadoc/2.4.0/).
+applications, so your application must use the [Apache Beam SDK for Java API version 2.4](https://beam.apache.org/documentation/sdks/javadoc/2.4.0/).
 
-## Before you execute
+## Before you run the installation program
```
Comment (pgerv12) [inline docs/beamrunner/beamrunner-2a-onprem.md]: I'm wondering if we could move this content to the "Before you start" section above and remove this header. Thoughts?

File: docs/beamrunner/beamrunner-2a-onprem.md [hunk 4]
```diff
@@ -18,68 +18,69 @@ Installing IBM® Streams Runner for Apache Beam from an installation of IBM Stre
...
 installation directory and entering the following command:
 ```bash
 . bin/streamsprofile.sh
 ```
 
 ## Installing and configuring Streams Runner
 
-1. Navigate to a directory where you have permission to create files.
-1. Extract the toolkit by entering the following command:
+1. Go to to a directory where you have permission to create files.
```
Comment (pgerv12) [inline docs/beamrunner/beamrunner-2a-onprem.md]: get rid of one of the 'to'

File: docs/beamrunner/beamrunner-2a-onprem.md [hunk 5]
```diff
@@ -18,68 +18,69 @@ Installing IBM® Streams Runner for Apache Beam from an installation of IBM Stre
...
 ```bash
-tar -zxvf $STREAMS_INSTALL/etc/beam/com.ibm.streams.beam-x.y.z.tar.gz
+tar -zxvf $STREAMS_INSTALL/etc/beam/com.ibm.streams.beam-<runner-version>.tar.gz
 ```
-1. (Optional) Create the Streams Runner environment variables. Although the variables are not required, the documentation refers to them for convenience. If you do not set the environment variables, you must use the full paths when you run the sample applications.
-    1. Navigate to the `samples` directory in the expanded toolkit, and set up environment variables for the runner:
+1. (Optional) Create the Streams Runner environment variables.
+
+    Tip: Although the variables are not required, the documentation refers to them for convenience. If you do not set the environment variables, you must use the full paths when you run the sample applications.
+    1. Go to the `samples` directory in the expanded toolkit and run the `streams-runner-env.sh` command to set up environment variables for the runner:
```
Comment (pgerv12) [inline docs/beamrunner/beamrunner-2a-onprem.md]:
```
Go to the `examples` directory...
```

File: docs/beamrunner/beamrunner-2b-sas.md [hunk 6]
```diff
@@ -31,58 +31,82 @@ Before you can download Streams Runner, you must have a Streaming Analytics serv
...
 ```
-1. (Optional) Configure the environment variables. Although the variables are not required, the documentation refers to them for convenience. If you do not set the environment variables, you must use the full paths when you run the sample applications.
-    1. Navigate to the `samples` directory in the expanded toolkit, and set up environment variables for the runner:
+1. (Optional) Configure the IBM Streams Runner environment variables.
+
+    Tip: Although the variables are not required, the documentation refers to them for convenience. If you do not set the environment variables, you must use the full paths when you run the sample applications.
+    1. Go to the `examples/bin` directory in the expanded toolkit and run the `streams-runner-env.sh` command to set up  environment variables for the runner:
     ```bash
-    cd com.ibm.streams.beam-1.1.1/samples
+    cd com.ibm.streams.beam-<runner-version>/examples/bin
```
Comment (pgerv12) [inline docs/beamrunner/beamrunner-2b-sas.md]: Get rid of bin here otherwise change next line to remove bin.

File: docs/beamrunner/io.md [hunk 7]
```diff
@@ -7,19 +7,20 @@ weight:  10
...
 prev:
-  file: wordcount
-  title: WordCount sample app
+  file: using
+  title: Using the runner
 next:
-  file: objstor
-  title: Using IBM Cloud Object Storage
+  file: sample
+  title: TemperatureSample app
```
Comment (pgerv12) [inline docs/beamrunner/io.md:14]: I thought I saw somewhere that used `Temperature sample app` elsewhere.

File: docs/beamrunner/io.md [hunk 8]
```diff
@@ -76,21 +77,25 @@ To use the storage service from Beam applications, you must specify the IBM Clou
...
+- `--awsCredentialsProvider`
 
-Beyond the service endpoint, the credentials for the service must be provided. The `--awsCredentialsProvider` option must be specified as a JSON format with a required `@type` field and `AWSCredentialsProvider` class as the value. It is recommended to use the [AWSStaticCredentialsProvider](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AWSStaticCredentialsProvider.html) along with the environment variables specified.
+    The credentials for the service must be provided. The `--awsCredentialsProvider` option must be specified as a JSON format with a required `@type` field and `AWSCredentialsProvider` class as the value. It is recommended to use the [AWSStaticCredentialsProvider](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AWSStaticCredentialsProvider.html) class along with the environment variables specified. pam - why, ie, get rid of "recommendation".
 
-Lastly, since Streams Runner does not include S3 and AWS libraries in its installation, the jars must be specified in the `--jarsToStage` option.
+- `--filesToStage`
```
Comment (pgerv12) [inline docs/beamrunner/io.md]: jarsToStage?

File: docs/beamrunner/messagehub.md [hunk 9]
```diff
@@ -1,71 +1,55 @@
 ---
 layout: docs
-title:  Using the IBM Cloud Message Hub service with IBM Streams Runner for Apache Beam
-navtitle: Using IBM Cloud Message Hub
-description:  Beam applications can produce/consume messages to/from IBM Cloud Message Hub using Beam's native KafkaIO.
+title:  io sample application for IBM Streams Runner for Apache Beam
```
Comment (pgerv12) [inline docs/beamrunner/messagehub.md]: Shouldn't these be capitalized?

**Review by pgerv12 — COMMENTED** [review]
Couple of requests/questions.

File: docs/beamrunner/beamrunner-2a-onprem.md (hunk 3 above)
Comment (pamlane) [reply docs/beamrunner/beamrunner-2a-onprem.md]: Totally agree!

File: docs/beamrunner/io.md (hunk 7 above)
Comment (pamlane) [reply docs/beamrunner/io.md:14]: I searched a bunch of different ways, and couldn't find one.

Comment (pgerv12) [reply docs/beamrunner/io.md:14]: Oops, it was TemperatureSample sample app (see _data/custom-sidenavs.yaml)

File: docs/beamrunner/beamrunner-2a-onprem.md (hunk 5 above)
Comment (pamlane) [reply docs/beamrunner/beamrunner-2a-onprem.md]: There were a lot of these. I changed all the ones that were part of a path, but didn't change ones that looked like part of a file name. Can you pay special attention to  these when you look at the next set of changes, to make sure I got this right?

**Review by pgerv12 — APPROVED** [review]

**Pull request merged**